pkg/controller/gittrack/handler_test.go

```diff
@@ -704,6 +729,24 @@ var _ = Describe("Handler Suite", func() {
 			Context("when the ClusterGitTrack is updated", func() {
 				AssertGitTrackUpdated(kind)
 			})
+
+			Context("when the ClusterGitTrack is excluded from managing namespaced object", func() {
+				BeforeEach(func() {
+					r.clusterGitTrackMode = farosflags.CGTMExcludeNamespaced
+					gto = testutils.ExampleGitTrackObject.DeepCopy()
+					gto.SetName("deployment-nginx")
+				})
+				AssertClusterGitTrackIgnoresNamespaced(&result)
+			})
+
+			Context("when ClusterGitTrack handling is disabled", func() {
```

Comment:
I think this isn't needed as the Handler should never be called when GitTrack handling is disabled I think, these tests will have to move (maybe later) to the actual reconciler tests once they are written I think